docs/README.md

```diff
@@ -522,6 +522,35 @@ randomSatoshis = true
 
 <!-- tabs:end -->
 
+## disableEnforceFocus
+
+> **The ‘disableEnforceFocus’ parameter is passed to the Dialog material UI component. Setting it to false can help with accessibility with technology such as screen readers. **
+
+?> This parameter is optional. Default value is true. Possible values are true or false.
+
+**Example:**
+<!-- tabs:start -->
+
+#### ** HTML **
+
+```html
+disableEnforceFocus="false"
```

Comment:
For HTML, we have to use lower case (`disable-enforce-focus` or something). I'm not sure how this works exactly as in it might automatically do this on build based on the camelCase... really not sure.

Reply:
I think it's a copy-paste error from me. I'll fix it up

Reply:
This is still an issue CTRL+F on `disableEnforceFocus` and change them to `disable-enforce-focus`. Clicking the sidebar link should take you to this section in the docs.

Reply:
Will need to update all 3 versions of the README.

Reply:
Can you recheck the latest version of the README's? The HTML parameter is correct in all of them for me.

Reply:
Maybe you didn't push? It's still not right here on GitHub or in the version I pulled.

https://github.com/PayButton/paybutton/pull/78/files

Reply:
When I click the link it shows the HTML version being correct. I'll double check with a clean pull

Reply:
Ah, are you talking about ## disableEnforceFocus on line 554? The original comment was on the html parameter which has been corrected